src/egl/drivers/dri2/platform_android.c

```diff
@@ -222,6 +224,28 @@ droid_window_dequeue_buffer(struct dri2_egl_surface *dri2_surf)
    return EGL_TRUE;
 }
 
+static int
+droid_resolve_format(struct dri2_egl_display *dri2_dpy,
+                     struct ANativeWindowBuffer *buf)
+{
+   int format;
+   int err;
+
+   if (buf->format != HAL_PIXEL_FORMAT_IMPLEMENTATION_DEFINED)
+      return buf->format;
+
+   if (!dri2_dpy->gralloc->perform)
+      return -1;
+
+   err = dri2_dpy->gralloc->perform(dri2_dpy->gralloc,
```

Comment:
It is you can find the API here: https://github.com/01org/minigbm/blob/gralloc1.0-rebase/cros_gralloc/gralloc1/cros_gralloc1_module.cc